roles/adoption_osp_deploy/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Prepare undercloud enviornment
+  ansible.builtin.import_tasks: prepare_undercloud.yml
+
+- name: Generate configuration files
+  ansible.builtin.import_tasks: config_files.yml
+
+- name: Group undercloud tasks
+  vars:
```

Comment:
so if you're able to go with `tags: undercloud` instead, that would make things easier.

Reply:
done